velox/common/base/Exceptions.h

```diff
@@ -65,8 +65,11 @@ template <typename Exception, typename StringType>
   static_assert(
       !std::is_same_v<StringType, std::string>,
       "BUG: we should not pass std::string by value to veloxCheckFail");
-  LOG(ERROR) << args.file << args.line << args.function << args.expression << s
-             << args.errorSource << args.errorCode;
+  LOG(ERROR) << "FilePath: " << args.file << ":" << args.line
```

Comment:
This is printing file path and line number, hence, FilePath is not exactly accurate. Consider "Line: ".